Description: link with needed libmagic
 required when building with ld --as-needed
Author: Julian Taylor <jtaylor@ubuntu.com>
Reviewed-by: Sebastian Reichel <sre@debian.org>
Debian-Bug: http://bugs.debian.org/653873
Forwarded: yes
Last-Update: 2013-12-29

--- a/libr/core/Makefile
+++ b/libr/core/Makefile
@@ -10,7 +10,7 @@
 OBJS+=anal.o project.o gdiff.o asm.o rtr.o vmenus.o disasm.o patch.o bin.o log.o
 
 include ../db/build.mk
-LDFLAGS+=${DL_LIBS}
+LDFLAGS+=${DL_LIBS} -lmagic
 
 ifeq ($(shell uname),OpenBSD)
 LDFLAGS+=-lpthread
